### PR TITLE
Update com.redhat.microprofile.ls pom for Nexus staging suite

### LINKED ISSUE #226: com.redhat.microprofile.ls cannot be released to JBoss Nexus
While trying to release to JBoss Nexus, I found out the release is refused because the POM is invalid:

- Project description missing
- License information missing
- SCM URL missing
- Developer information missing

Also we should add the Nexus Staging Maven Plugin configuration

### PR DESCRIPTION
**Pull request by jeffmaury**
Fixes #226

Signed-off-by: Jeff MAURY <jmaury@redhat.com>

### CODE REVIEW

File: microprofile.ls/com.redhat.microprofile.ls/pom.xml [hunk 1]
```diff
@@ -7,7 +7,30 @@
 	<version>0.0.6-SNAPSHOT</version>
 
 	<name>MicroProfile Language Server</name>
+	<description>MicroProfile Language Server</description>
 	<url>http://maven.apache.org</url>
```
Comment (angelozerr) [inline microprofile.ls/com.redhat.microprofile.ls/pom.xml]: @fbricon what is this url?

Comment (fbricon) [reply microprofile.ls/com.redhat.microprofile.ls/pom.xml]: yes absolutely

Comment (fbricon) [reply microprofile.ls/com.redhat.microprofile.ls/pom.xml]: well no, https://github.com/redhat-developer/quarkus-ls for now

Comment (angelozerr) [reply microprofile.ls/com.redhat.microprofile.ls/pom.xml]: @fbricon thanks for your feedback. @jeffmaury could you change that please in your PR please.

**Comment by angelozerr** [comment]
@fbricon could you review this PR please. Thanks

File: microprofile.ls/com.redhat.microprofile.ls/pom.xml [hunk 2]
```diff
@@ -87,6 +111,16 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
```
Comment (fbricon) [inline microprofile.ls/com.redhat.microprofile.ls/pom.xml]: please fix the formatting

**Pull request merged**